.code-samples.meilisearch.yaml

```
@@ -192,6 +192,22 @@ update_displayed_attributes_1: |-
   });
 reset_displayed_attributes_1: |-
   client.index("movies").resetDisplayedAttributesSettings();
+get_typo_tolerance_1:
+  client.index("books").getTypoToleranceSettings();
+update_typo_tolerance_1: |-
+  TypoTolerance typoTolerance = new TypoTolerance();
+  typoTolerance.setDisableOnAttributes(new String[] {"title"});
+  HashMap<String, Integer> minWordSizeTypos =
+          new HashMap<String, Integer>() {
+              {
+                  put("oneTypo", 4);
+                  put("twoTypos", 10);
+              }
+          };
+  typoTolerance.setMinWordSizeForTypos(minWordSizeTypos);
+  client.index("movies").updateTypoToleranceSettings(typoTolerance);
```

If you disagree is ok, but you at least need to check the index name :)

```suggestion
  client.index("books").updateTypoToleranceSettings(typoTolerance);
```